src/components/Tables/DataTable/fragments/inputs/TableInputMask.vue

```diff
@@ -0,0 +1,32 @@
+<template>
+  <InputMask
+    :mask="preset ? presets[preset.toLowerCase()].mask : undefined"
+    :placeholder="
+      preset && edit
+        ? presets[preset.toLowerCase()].mask?.replace(/9/g, '_')
+        : undefined
+    "
+    unmask
+    :readonly="!edit"
+    :class="edit ? '' : '!border-none !shadow-none !bg-transparent'"
+  />
+</template>
+
+<script setup lang="ts">
+import InputMask, { type InputMaskProps } from "primevue/inputmask";
+
+export type TableInputMaskProps = {
```

Comment:
```suggestion
type TableInputMaskProps = {
```
se precisar usar fora desse contexto, exporte-o em `types` e importe-o aqui.